### PR TITLE
Cels-Fahr Program 

### PR DESCRIPTION
**Pull request by pjbaranek**
Program for converting Celsius to Fahrenheit

### CODE REVIEW

**Comment by pjbaranek** [comment]
Added the getchar program as well

**Comment by pjbaranek** [comment]
Added line counting program

**Comment by pjbaranek** [comment]
Added print input

**Comment by pjbaranek** [comment]
Added reverse string program

File: getchar.c [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+ /* verifying getchar output */
+int  main()
+{
+int c;
+if(c = getchar() != EOF){
+  printf("%d\n", c);
+  }
+return 0;
```
Comment (luminoctum) [inline getchar.c:9]: I find that the indentation is not consistent. In this gethcar program, you are using two spaces for indentation, while in the previous program, you are using 4 spaces for indentation. Make that spacing is consistent in the future. You can use `tab` for indentation and it should automatically be 2 spaces.

**Review by luminoctum — APPROVED** [review]
Good to go expect for the indentation. I'll accept them and you can fix the indentation next time.

**Pull request merged**